src/Admin/KeycloakClient.php

```diff
@@ -271,6 +271,7 @@
  * @method array addUserToGroup(array $args = array()) { @command Keycloak addUserToGroup }
  * @method array deleteUserFromGroup(array $args = array()) { @command Keycloak deleteUserFromGroup }
  * @method array resetUserPassword(array $args = array()) { @command Keycloak resetUserPassword }
+ * @method array getUserSessions(array $args = array()) { @command Keycloak resetUserPassword }
```

Comment:
please update the `@command` to `getUserSessions`

Reply:
Thanks for seeing this omission. Sorry. It's corrected.